nautobot/core/tables.py

```diff
@@ -34,7 +35,9 @@ class Meta:
             "class": "table table-hover table-headings",
         }
 
-    def __init__(self, *args, user=None, hide_hierarchy_ui=False, **kwargs):
+    def __init__(
+        self, *args, table_changes_pending=False, saved_view_pk=None, user=None, hide_hierarchy_ui=False, **kwargs
```

Comment:
Would it be cleaner to pass around the `SavedView` instance instead of just its PK?